output/Form.md

```diff
@@ -9,11 +9,22 @@ title: Yerun Open Science Award form
 -   [*Introduction*]{.underline}*: short overview of the initiative. What are
     its key Open Science objectives (and others, if applicable)? Who are the
     target groups? What are the main activities organised?*
+    
+Our initiative consists of developing an open science framework that can be used across the research projects in our lab and exported to other departments, both in and outside our institution. It consists of three steps: data data management, which will be done using OPAL, part of the Obiba environment; data computing, which will follow the literate computing framework using Markdown and R Studio; and version control, which will be done using Git. EXPAND. Our key Open Science objective is to increase reproducibility and to engage the research community in our projects. Our main target group is all researchers worldwide and ultimately the entire population.
+
 -   [*Context*]{.underline} *in which the initiative was implemented: What
     prompted you to develop this initiative?*
+    
+As tools like Open Science Framework (OSF) rapidly grow, world-leading funding bodies, such as Welcome Trust in the United Kingdom and the National Institute of Health in the United States of America, include compulsory open access guidelines for their research projects. More recently, the European Commission started to adhere to this initiative, and calls such as the Horizon 2020 programme for research and innovation include specific open access policies. Our lab is a World Health Organisation collaborating centre focused on mental health services research and training. We are based at the Department of Psychiatry at the Universidad Autónoma de Madrid, which is or has been involved in nearly ten projects funded by the European Union. In this initiative, we used our expertise in two of these projects, namely SYNCHROS and RESPOND, with the broad aim to expand it to other projects.
+
+With SYNCHROS we used literate computing and version Control, with RESPOND we are using the three steps.
+    
+
 -   [*Obstacles and solutions*]{.underline}*: What challenges, if any, were
     encountered? Were they/how were they solved?*
 
+We encountered obstacles both within and outside our lab. In our lab, the main challenge was overcoming the lack of specific training in OS. We are currently doing Tidyverse sessions, we bought a server, we are trying to use Git, etc. Outside the institution, the main challenge is that other partners see this as an investment they don’t have time to make.
+
 <!--# blank field, max 400 words -->
```

Comment:
Please note: There is only **ONE blank field** for this point, so all the content (addressing the three bullet points above) should go in a single place (400 words max.)